android/src/main/java/nl/fhict/happynews/android/PostManager.java

```diff
@@ -79,6 +79,49 @@ public void onCompleted(Exception e, Page result) {
             });
     }
 
+
+    /**
+     * Sends content of the first page to the FeedAdapter.
+     * @param context The Application context.
+     * @param listener Implementing the LoadListener Interface.
+     */
+    public void refresh(Context context, final LoadListener listener) {
```

Comment:
I think this method and `#loadPage(int page, int size, Context context, final LoadListener listener)` look very similar. Is there any way we can merge these two?